packages/gatsby-image/src/index.js

```diff
@@ -194,13 +214,14 @@ class Image extends React.Component {
 
     const imagePlaceholderStyle = {
       opacity: this.state.imgLoaded ? 0 : 1,
-      transitionDelay: `0.25s`,
+      transitionDelay: this.state.imgLoaded ? `0.5s` : `0.25s`,
```

Comment:
when transitioning a `critical` image to opaque:1 - Chrome 67/68 flashes/flickers - not certain if there's a reason to synchronize the placeholder while fading in the real image.